src/main/java/MovieInformation/controller/MovieInformationController.java

```diff
@@ -0,0 +1,42 @@
+package MovieInformation.controller;
+
+import MovieInformation.MovieInformationNotFoundException;
+import MovieInformation.entity.Movie;
+import MovieInformation.service.MovieInformationService;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class MovieInformationController {
+    private final MovieInformationService movieInformationService;
+
+    public MovieInformationController(MovieInformationService movieInformationService) {
+        this.movieInformationService = movieInformationService;
+    }
+
+    //GETの実装
+    //全件取得の実装
+    @GetMapping("/movie-information")
```

Comment:
こういうパス設計にするのはどうでしょうか？
```suggestion
    @GetMapping("/movies")
```

Reply:
273dff7846e0532e5c5fa6833dfc4df07d11a4bd
パス設計を修正しました！
確かにシンプルでわかりやすいです。